xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleStream.cpp

```diff
@@ -73,7 +72,7 @@ bool CDVDSubtitleStream::Open(const std::string& strFile)
         totalread += read;
     } while (read > 0);
 
-    delete pInputStream;
+    pInputStream.reset();
```

Comment:
On 2016/03/24 21:33, Garrett Brown <notifications@github.com> wrote:
> > @@ -73,7 +72,7 @@ bool CDVDSubtitleStream::Open(const std::string& strFile)
> >          totalread += read;
> >      } while (read > 0);
> >  
> > -    delete pInputStream;
> > +    pInputStream.reset();
> 
> returning should automatically reset, correct? this line can be dropped

Yes, but the old code did it here, so I did the same, to retain old
behavior.  Sometimes, there can be an advantage in freeing an object
earlier in the scope, and in this case, I wasn't sure if that early
delete was intentional.  Of course, I'll drop it if you prefer so.